app/controllers/application_controller.rb

```diff
@@ -5,6 +5,157 @@ class ApplicationController < Sinatra::Base
   configure do
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).